css/index.css

```diff
@@ -137,10 +141,24 @@ footer nav {
     justify-content: space-between;
     align-items: center;
     padding: 20px 2%;
-    font-size: 14px;
+    font-size: 14px;    
+    
+}
+header footer nav a{
```

Comment:
display: inline-flex and display: flex are very similar, there is really only one difference display: inline-flex does not make flex items display inline.